src/App.js

```diff
@@ -1,12 +1,35 @@
-import './App.css';
+import "./App.css";
+import React, { useState } from "react";
+import Counter from "./components/Counter";
+import SearchForm from "./components/SearchForm";
+import GenreSelect from "./components/GenreSelect";
 
 function App() {
```

Comment:
as all component are arrow function, here also we can make:
```suggestion
const App = () => {
```